### DISCUSSION TITLE
Add a few new video cards

### DISCUSSION THREAD

**Discussion by Rasthro** [86Box/86Box#2792] [Ideas]
Hi, I want to suggest some NEW video cards 86box NEVER EVER thought of adding it in their project!

Here's the list of video cards that 86box should have:
Diamond SPEA Mirage Video (Diamond Stealth24)
Trident TVGA 9000c
Trident TVGA 9000i
VooDoo 3 3500 (It's just slightly better than VooDoo 3 3000, I'm sure our PC can emulate this)
Tseng ET3000AX
Tseng ET6000
ATI Mach32

**Comment by TC1995** [comment]
provide documents of the et6000 please.

**Comment by Rasthro** [comment]
Bruh, I am not asking to add VooDoo 4/5, How can you say VooDoo 3 3500 is impossible to emulate, Already VooDoo 3 3500 is not very powerful, Ofc it’s a little better than VooDoo 3 3000, Our PC CAN EMULATE VOODOO 3 3500!!! YOU CANNOT TELL IT CANT!!! Sent from Mail for Windows From: JimboMumboSent: Sunday, November 6, 2022 9:52 AMTo: 86Box/86BoxCc: Rasthro; AuthorSubject: Re: [86Box/86Box] Add a few new video cards (Discussion #2792) voodoo3_spec_1401363489.pdfScans-00157826.pdfI hope it helps.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

**Comment by DDXofficial** [comment]
> Bruh, I am not asking to add VooDoo 4/5, How can you say VooDoo 3 3500 is impossible to emulate, Already VooDoo 3 3500 is not very powerful, Ofc it’s a little better than VooDoo 3 3000, Our PC CAN EMULATE VOODOO 3 3500!!! YOU CANNOT TELL IT CANT!!!

Based on your reaction here, you clearly have no idea of how complex it is to emulate video cards or any form of hardware. I'm no developer myself, but I can tell you that adding a video card to 86Box and ensuring it works is not as easy as just writing a few lines of code. Someone would need to have physical access to said video card OR it would need to be well documented (something similar to what @JimboMumbo posted earlier). Just because a piece of hardware is "not very powerful" as you say it is doesn't necessarily mean it can be emulated if there's no documentation on it whatsoever.

> some NEW video cards 86box NEVER EVER thought of adding it in their project!

Chances are we thought of adding it and/or it was requested in the past, but either no one in the community had said video card or it had zero documentation available online.

That attitude of yours isn't exactly helping, either.
